src/frontend/src/flows/authenticate/validateDerivationOrigin.ts

```diff
@@ -17,18 +17,24 @@ export const validateDerivationOrigin = async (
   }
 
   // check format of derivationOrigin
-  const matches = /^https:\/\/([\w-]*)(\.raw)?\.ic0\.app$/.exec(
+  const matches = /^https:\/\/([\w-]+)(\.raw)?\.ic0\.app$/.exec(
```

Comment:
unrelated to this PR, but shouldn't `(\.raw)` be a non-capturing group? or is that used somewhere?

Reply:
just checking, no need! thanks for the clarification